raster-foundry/raster-foundry
# Log healthcheck services that are failing

## Overview

Sometimes our tile service goes down but we don't always know the cause of the failure. This is to help log the reason of the failure.

Before this - we were logging the quota healthcheck but not any other failures.

### Checklist



## Testing Instructions

- Rebuild servers and start them
- Make a request to the healthcheck - should work: `http :8081/healthcheck`
- Turn off memcached: `docker-compose stop memcached`
- Make another request to the healthcheck: `http :8081/healthcheck` . This one should fail (it might be a 500 the first time, so make another one. Then note there is a message in the logs about the failing service.
- Make another request to the healthcheck, this time with a header for the trace id: `http :8081/healthcheck 'X-Amzn-Trace-Id:Test123'` and this time the log message should include a trace id.

Closes #5069
